src/resources/app-en.json

```diff
@@ -511,5 +513,50 @@
   "textInputOptionDescription": "The respondent will be able to enter a text response",
   "textInputOptionRequired": "Required",
   "minValue": "Min value",
-  "maxValue": "Max value"
+  "maxValue": "Max value",
+  "mdEditorBold": "Bold",
```

Comment:
I think it's better to use the key without mdEditor prefix, there could be cases that have the same value, for example just bold

Reply:
done